src/GuestConfiguration/GuestConfiguration/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.10.8
```

Comment:
no need to update version, please move message under "upcoming release" section

Reply:
updated.